plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -167,18 +168,16 @@ global proc int mayaUsdTranslatorImport (string $parent,
             {
                 // Hide the field (and shrink it) and show the button (which will align with label).
                 textField -e -vis false -w 1 mayaUsdTranslator_SelectFileField;
+                button -e -visible true -w $bw mayaUsdTranslator_ViewHierBtn;
```

Comment:
The previous code below has a button "exists" test. And we have a exists test for the textfield. I guess we should keep that test here and line 177?

Reply:
I started a comment that explained this but maybe I didn't actually leave it here.  I didn't bother with the extra "exists" check because the logic that decides if this UI should be created or not creates both of them so if one exists then the other is guaranteed to be there as well.

Reply:
Okay looks good then.